Frameworks/include/NSRunLoop+Internal.h

```diff
@@ -31,7 +31,7 @@
 - (void)_removeInputSource:(NSInputSource*)source forMode:(NSString*)mode;
 - (void)_addObserver:(NSObject*)observer forMode:(NSString*)mode;
 - (void)_removeObserver:(NSObject*)observer forMode:(NSString*)mode;
-- (StrongId<NSArray*>)_statesForMode:(NSString*)mode;
+- (StrongId<NSArray>)_statesForMode:(NSString*)mode;
```

Comment:
Actually, it could be a better solution to simply make that return an autoreleased thing, and drop the StrongId from each of its consumers. Feel free to go either way!